docs/synthetic-tokens/long-short-pair.md

```diff
@@ -71,6 +74,10 @@ Disputers can refute a price submitted by a Proposer within the proposal livenes
 
 ![](/docs/lsp-tokens/lsp_settle.png)
 
+### `requestEarlyExpiration`
+
+The `enableEarlyExpiration` parameter can be set to `true` on contract deployment which enables the LSP contract to request to be settled early by calling the Optimistic Oracle. When an LSP requests an early expiration, a price request is initiated to the optimistic oracle at the provided timestamp with a modified version of the ancillary data that includes the key `earlyExpiration:1`. This signals to the Optimistic Oracle that this is an early expiration request, rather than a standard settlement.
```

Comment:
We should include more information here. Specifically:
- Anyone can call this.
- proposers can return the actual settlement value, or they can return `type(int256).min` (-57896044618658097711785492504343953926634992332820282019728792003956564819968) to indicate that the contract is not settleable at this time. If the latter is returned, then the contract does not settle at that time.

Reply:
I updated to include the above points. Anything else you think is missing?

Reply:
Nope, think that is it.